atlascope/urls.py

```diff
@@ -33,7 +30,6 @@
 
 urlpatterns = [
     path('accounts/', include('allauth.urls')),
-    path('oauth/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
Perhaps we should rip out `oauth` entirely, since this is about simplification. At this point, we can remove `oauth2_provider` and `guardian` from `INSTALLED_APPS` in our settings so we don't end up migrating the db to support these things. Then we can also remove them from our dependencies.

Reply:
I agree in principle but also it won't do any harm to leave this stuff in place. The goal, more than simplification, is an easier or simplified developer experience. I looked into ripping these things out by the root and it seems like more trouble than it's worth (other than guardian, which I will do; stand by).

Reply:
See 142f37f7e86129df41dedef41bda1f42b6727b92.